Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/Shadow/DirectionalLightShadow.azsli

```diff
@@ -420,3 +459,12 @@ float3 DirectionalLightShadow::AddDebugColoring(
     }
     return color;
 }
+
+float DirectionalLightShadow::CalculateCascadeBlendAmount(const float3 texCoord)
+{
+    const float3 distanceToOne = float3(1.0f - texCoord.x, 1.0f - texCoord.y, 1.0f - texCoord.z);
```

Comment:
super nit: 
for ease of read can be  'const float3 distanceToOne = 1 - texCoord'

Reply:
Thanks, changed